---
title: "Marco Tello-Palencia, MSc"
description: "Bioinformatician & Data Scientist"
lead: ""
date: 2023-09-07T16:33:54+02:00
lastmod: 2023-09-07T16:33:54+02:00
draft: false
seo:
  title: "Marco Tello-Palencia Bioinformatics Home" # custom title (optional)
  description: "" # custom description (recommended)
  canonical: "" # custom canonical URL (optional)
  noindex: false # false (default) or true
---

<div style="text-align: center;">
  <a href="mailto:mtello@student.ubc.ca">mtello@student.ubc.ca</a> |
  <a href="https://linkedin.com/in/marco-tello-palencia">LinkedIn</a> |
  <a href="https://github.com/mtello22">GitHub</a>
</div>

## PROFESSIONAL SUMMARY
Bioinformatician with 6 years of experience in analyzing genomic, transcriptomic, and methylome datasets. Looking for an opportunity to continue developing machine learning models to extract biological insights from multi-omic datasets.

## SKILLS

<div style="display: flex; flex-wrap: wrap; justify-content: space-between;">
  <div style="flex: 1; padding: 10px;">
    <ul>
      <li><strong>Programming:</strong> Python, R, mySQL, bash.</li>
      <li><strong>Machine Learning:</strong> scikit-learn, SHAP value analysis.</li>
      <li><strong>Report and Documentation:</strong> GitHub, R-markdown, Jupyter-notebook.</li>
      <li><strong>High-performance Computing:</strong> PBS, NextFlow.</li>
      <li><strong>Visualization:</strong> ggplot2, R-Shiny apps, Tableau.</li>
    </ul>
  </div>
  <div style="flex: 1; padding: 10px;">
    <ul>
      <li><strong>Bioinformatic Pipelines:</strong>
        <ul>
          <li>Quality Control of Sequencing Data</li>
          <li>Differential Expression using RNA-seq</li>
          <li>Analysis of Whole Genome Bisulfite Sequencing</li>
          <li>Gene Set Enrichment Analysis</li>
          <li>Gene Regulatory Network Inference</li>
          <li>Single-cell RNA-seq</li>
        </ul>
      </li>
    </ul>
  </div>
</div>

## EXPERIENCE

### Independent Project – Identification of melanoma resistant factors in scRNA-seq samples
Nov 2023 – Feb 2024
[GitHub Repo](https://github.com/your_repo)
- Performed quality control and pre-processing of single-cell RNA-seq data using custom R scripts.
- Trained XGBoost Classifier using scikit-learn environment to predict tumor resistance, achieving AUPR = 0.91 in held-out samples.
- Generated visualizations using R Shiny apps to facilitate data exploration and t-SNE visualization.

### Bioinformatician – Korthauer Lab, UBC
Jan 2020 – Dec 2023
Prediction of gene expression associated with healthy cell type differences.
- Designed supervised learning models to regress gene expression by integrating multi-omic datasets.
- Improved model interpretability to discover DNA methylation as the primary predictor of gene expression among 400 features in the trained models.
- Designed strategies for training and evaluation of multiple supervised regression models.

### Bioinformatician – Stefanska Lab, UBC
Jun 2021 – Aug 2021; Feb 2024 – Current
[GitHub Repo](https://github.com/your_repo)
Analysis of transcriptomic differences in blood for rats with different  supplemented diets (Manuscript in preparation).
- Implemented new protocols for bioinformatic analysis, resulting in the identification of 30+ differentially expressed genes.

### Bioinformatician – Deplancke Lab, EPFL
Jan 2019 – Aug 2019
Identification of epigenetic signatures during human adipocyte differentiation.
- Performed quality control and pre-processing of 20+ raw genotyping and ChIP-seq samples.

### Research Assistant – Freyre Lab, UNAM
Feb 2017 – Dec 2018
*De novo* inference of gene regulatory network for *Neurospora crassa*.
- Implemented ensemble approach to integrate 3 different inferred regulatory networks for *S. cerevisiae* and *N. crassa* based on 50+ public transcriptomic datasets.
- Identified biological functions associated with subnetworks controlled by circadian regulators.

## TEACHING AND MENTORING EXPERIENCE
- **Mentor URO-REX Program (6 mentees) – Korthauer Lab, UBC**
  Oct 2021 – Apr 2022
  - Managed team to formulate, perform, and communicate results from a bioinformatics research project.
- **TA STAT540: Statistical Methods for High-Dimensional Biology – UBC**
  Jan 2021 – Apr 2022
  - Updated and imparted bi-weekly tutorials on specific bioinformatic pipelines and tools.
- **Mentor Systems Biology Workshop (11 mentees) – Freyre Lab, UNAM**
  Jun 2017 – Nov 2018
  - Benchmark of methods for the inference of gene regulatory networks. (Published)
- **TA Programming Principles and Scientific Computation – UNAM**
  Jun 2017 – Nov 2018
  - Developed exercises and activities to teach algorithm design to first-year undergraduate students.

## EDUCATION
**MSc in Bioinformatics, University of British Columbia**
Feb 2024

**BSc in Genomic Sciences, Universidad Nacional Autónoma de México**
May 2018

## PUBLICATION AND COMPETITIVE SCHOLARSHIPS

Escorcia-Rodríguez, J. M., Gaytan-Nuñez, E., Hernandez-Benitez, E. M., Zorro-Aranda, A., **Tello-Palencia, M.A.**, & Freyre-González, J. A. (2023). Improving gene regulatory network inference and assessment: The importance of using network structure. Frontiers in Genetics, 14, 143382.

**BC Research Institute for Children's and Women's Health Graduate Studentship**
Sept 2021 – Aug 2022

**NSERC-CREATE Scholarship Awardee**
Sept 2020 – Aug 2021
